definitions/checks/check_epel_repository.rb

```diff
@@ -0,0 +1,19 @@
+class Checks::CheckEpelRepository < ForemanMaintain::Check
+  metadata do
+    label :check_epel_repository
+    description 'Check if EPEL repoistory enabled on system'
+    tags :pre_upgrade
+    confine do
+      feature(:downstream)
+    end
+  end
+
+  def run
+    assert(!epel_enabled?, 'System is subsribed to EPEL repository')
```

Comment:
Since the `yum repolist` can take a bit of time, especially when registered with subscription manager, I would recommend using spinner while waiting:

```ruby
with_spinner("Checking for presence of EPEL repository") do
  assert(!epel_enabled?, 'System is subsribed to EPEL repository')
end
```